wormhole-connect/src/utils/index.ts

```diff
@@ -462,3 +462,15 @@ export const millisToRelativeTime = (ts: number): string => {
     return '~1 minute ago';
   }
 };
+
+export const formatDuration = (seconds: number) => {
```

Comment:
Thought bubble: The number of time formatting utilities is growing :) I think we just need dayjs:
https://day.js.org/docs/en/display/format

Reply:
Good idea, I'll open an issue for it since the scope is wider than this targeted PR